packages/web/src/client/components/Links/InlineLink.tsx

```diff
@@ -23,9 +23,4 @@ const Anchor = styled(LinkBase, {
   fontWeight: 'inherit',
   lineHeight: 'inherit',
   fontFamily: 'inherit',
-  textDecoration: 'none',
```

Comment:
maybe these should be `inherit` like the above? What do you think?

Reply:
@joshuaellis 

how would that work? so the new styling goes inside the `LinkBase` and the text decoration here should be `inherit`?
(I tried that btw, doesn't seem to work)

Reply:
No i just think set it as inherit incase we set it elsewhere.